src/components/authentication/FaceBookButton.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FacebookLogin from 'react-facebook-login/dist/facebook-login-render-props';
+import FacebookIcon from 'components/icons/Facebook';
+
+const FacebookButton = ({ onSuccessHandler }) => (
+  <FacebookLogin
+    appId="414927182376293"
+    autoLoad={false}
+    fields="name,email"
+    callback={onSuccessHandler}
+    render={renderProps => (
```

Comment:
This is going to try to submit the login form instead of redirecting to facebook, I remember we had fixed it by adding a type ="button" attribute

Reply:
yes we did, let me check why it still has that